content/authors/Carson-Chambers/_index.md

```diff
@@ -0,0 +1,49 @@
+---
+# Display name
+title: Carson Chambers
+
+# Username (this should match your github username)
+authors:
+- CarsonPC
+
+# Role/position e.g. Graduate student
+role: Graduate Student
+
+# Organization membership
+Organizations/Affiliations:
+organizations:
+- name: Pacific Institute for the Mathematical Sciences
+  url: https://m2pi.ca
+
+# A list of your interests, add as many as you want
+interests:
+- General Purpose Data Science
+- Mathematical Biology
+
+# Your educational history, add as many as you want
+education:
+  courses:
+  - course: BSc - Mathematics
+    institution: University of the Fraser Valley
+
+# Social/Academic Networking, delete any items you don't want to use
+social:
+- icon: github
+  icon_pack: fab
+  link: https://github.com/CarsonPC
+- icon: linkedin
+  icon_pack: fab
+  link: https://linkedin.com/in/carson-chambers-363767206/
+
+user_groups:
+- committee: A. Joshi, A. Khorasani, C. Chambers
```

Comment:
Hi Carson, We use the value of committee to distinguish between user groups, right now your profile isn't being identified in the right group. Could you add a commit to change your file to read `committee: members` e.g.
```
user_groups:
- committee: members
   role: Team 6 Project Member
```